scripts/run_tests.bat

```diff
@@ -0,0 +1,9 @@
+rem Copyright (c) Microsoft Corporation
+rem SPDX-License-Identifier: MIT
+@echo The script assumes install-ebpf.bat has already been run.
```

Comment:
ok, but maybe it should run install-ebpf.bat, and install-ebpf.bat should be a no-op if it's already installed.